doc/source/basics.rst

```diff
@@ -1418,29 +1418,48 @@ description.
 
 .. _basics.sorting:
 
-Sorting by index and value
---------------------------
+Sorting
+-------
+
+.. warning::
+
+   The sorting API as substantially changed in 0.17.0, see :ref:`here <whatsnew_0170.api_breaking.sorting>` for these changes.
+   In particular, all sorting methods now return a new object, and DO NOT operate in-place.
 
 There are two obvious kinds of sorting that you may be interested in: sorting
-by label and sorting by actual values. The primary method for sorting axis
-labels (indexes) across data structures is the :meth:`~DataFrame.sort_index` method.
+by label and sorting by actual values.
+
+By Index
+~~~~~~~~
+
+The primary method for sorting axis
+labels (indexes) are the :meth:`~Series.sort_index` and the :meth:`~DataFrame.sort_index` methods.
```

Comment:
can you remove the `~` here? because now it will be displayed as two times 'sort_index' without seeing the difference